bitnami/harbor/README.md

```diff
@@ -0,0 +1,348 @@
+# Helm Chart for Harbor
+
+This Helm chart has been developed based on [goharbor/harbor-helm](https://github.com/goharbor/harbor-helm) chart but including some features common to the Bitnami chart library.
+For example, the following changes have been introduced:
+
+- Possibility to pull all the required images from a private registry through the  Global Docker image parameters.
+- Redis and PostgreSQL are managed as chart dependencies.
+- Liveness and Readiness probes for all deployments are exposed to the values.yaml.
+- Uses new Helm chart labels formating.
+- Uses Bitnami container images:
+  - non-root by default
+  - published for debian-9, ol-7, and eventually centos-7
```

Comment:
Eventually centOS?

Reply:
Delete that note at the moment as not sure when it will be available.